### ISSUE TITLE
Implement the export process

### ISSUE THREAD

**Issue by nukeador** [Common-Voice/sentence-collector#27]
We need to document how the export process of the validated sentences works.

**Comment by nukeador** [comment]
The exported sentences should land in their locale folder in this repo as txt files:

https://github.com/mozilla/voice-web/tree/master/server/data/

**Comment by MichaelKohler** [comment]
This depends on #4 as we only want approved strings to be exported.

**Comment by MichaelKohler** [comment]
As discussed with Nuke, we're gonna export all the approved strings into sentence-collector.txt to make our life easier (not tracking chunks)

**Comment by MichaelKohler** [comment]
So currently I see the following tasks:

* Export approved sentences
* Integrate the possibility to do cleanups (at the beginning used for sorting the sentences to create better diffs)
* (?) Check for duplicate sentences already in the voice-web repo and not export them
* Create yarn run task to have a "one command" process
* Manually creating voice-web commits/PRs with the export for now
* Document all of this

**Comment by nukeador** [comment]
Let's aim to have a functional export fist and document how, the rest are nice to haves for this release.

**Comment by MichaelKohler** [comment]
PR: https://github.com/Common-Voice/sentence-collector/pull/133

export: https://github.com/mozilla/voice-web/compare/master...MichaelKohler:sc_export_test?expand=1
